booklet/src/sections/productionazing.tex

```diff
@@ -181,10 +180,9 @@ \subsection{Customizing the MOJO Settings}
 \begin{itemize}
     \item \texttt{predictionCol} - Specifies the name of the generated prediction column. Default value is \texttt{prediction}.
     \item \texttt{detailedPredictionCol} - Specifies the name of the generated detailed prediction column. The detailed prediction column, if enabled, contains additional details, such as probabilities, shapley values etc. The default value is \texttt{detailed\_prediction}.
-    \item \texttt{withDetailedPredictionCol} - Enables or disables the generation of the detailed prediction column. By default, it is disabled.
```

Comment:
@jakubhava Are you talking about this file? I thought that I remove all occurences of `withDetailedPredictionCol`, but maybe I missed something...

Reply:
Ah, sorry, my bad. I didn't mention on the first look that you already updated this file. All good then!